### PR TITLE
Support for iTwin Viewer

### PR DESCRIPTION
**Pull request by StefanRetief**
Support for the iTwin Viewer and flow for sample widgets.

### CODE REVIEW

**Comment by BaoTon** [comment]
Occasionally getting this error when switching samples:

image

**Comment by BaoTon** [comment]
Another issue caused by switching between samples:

image

File: src/frontend-samples/view-attributes-sample/ViewAttributesWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "common/samples-common.scss";
```
Comment (BaoTon) [inline src/frontend-samples/view-attributes-sample/ViewAttributesWidget.tsx:5]: These common/samples-common.scss no longer needs to be imported right?

**Review by BaoTon — COMMENTED** [review]
I think the WebGL Context is still not being properly cleaned up. Ocassionally getting memory issues:

image

Solved with close on IModelConnections

**Comment by williamkbentley** [comment]
I got this error when opening GlobalData sample
image

**Review by williamkbentley — APPROVED** [review]
Looks good to me.

I got WebGl error only once when I was changing samples rapidly. Like beyond reason rapidly.

**Comment by BaoTon** [comment]
> Looks good to me.
> 
> I got WebGl error only once when I was changing samples rapidly. Like beyond reason rapidly.

Same here, that's the biggest concern I have but else it also looks good on my end.

**Pull request merged**